Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1453,11 +1453,13 @@ section MeasureSpace
 
 variable {s : Set α} [MeasureSpace α] {p : α → Prop}
 
-instance Subtype.measureSpace : MeasureSpace (Subtype p) :=
-  { Subtype.instMeasurableSpace with
-    volume := Measure.comap Subtype.val volume }
+/-- In a measure space, one can restrict the measure to a subtype to get a new measure space. -/
```

Comment:
You write in the PR description:

> There are however other reasonable constructions, like the normalized restriction for probability measure, or the subspace measure when restricting to a measure. We disable the global instance `Subtype.measureSpace` to make these other choices possible.



Please put this or a version of it in the docstring too!